docs/resources/Guild.md

```diff
@@ -15,13 +15,12 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | discovery_splash              | ?string                                                                             | [discovery splash hash](#DOCS_REFERENCE/image-formatting); only present for guilds with the "DISCOVERABLE" feature                        |
 | owner?                        | boolean                                                                             | true if [the user](#DOCS_RESOURCES_USER/get-current-user-guilds) is the owner of the guild                                                |
 | owner_id                      | snowflake                                                                           | id of owner                                                                                                                               |
-| permissions?                  | integer                                                                             | legacy total permissions for [the user](#DOCS_RESOURCES_USER/get-current-user-guilds) in the guild (excludes overrides)                   |
-| permissions_new?              | string                                                                              | total permissions for [the user](#DOCS_RESOURCES_USER/get-current-user-guilds) in the guild (excludes overrides)                          |
+| permissions?                  | string                                                                             | total permissions for [the user](#DOCS_RESOURCES_USER/get-current-user-guilds) in the guild (excludes overrides)                   |
```

Comment:
Would it be possible to document _when_ we actually get this and when we don't? That would provide us with much more of an ability to use this.

From what I can tell, this is only returned by the get user guilds endpoint?

Reply:
the description says `the user` and links to the only endpoint where it is returned; it just doesn't make sense in any other situation

Reply:
IMO we might refactor this part of the doc in the future, but for now I think we should leave it as-is since its description is aligned with other parts of this table (e.g. `owner`)

I also got bamboozled by this btw XD